kas/anchor.go

```diff
@@ -66,6 +68,86 @@ func NewKASAnchor(kasConfig *KASConfig, db AnchorDB, bc BlockChain) *Anchor {
 	}
 }
 
+// AnchorPeriodicBlock periodically anchor blocks to KAS.
+// if given block is invalid, it does nothing.
+func (anchor *Anchor) AnchorPeriodicBlock(block *types.Block) {
+	if !anchor.kasConfig.Anchor {
+		return
+	}
+
+	if block == nil {
+		logger.Error("KAS Anchor : failed to get a block form the event")
+		return
+	}
+
+	if block.NumberU64()%anchor.kasConfig.AnchorPeriod != 0 {
+		return
+	}
+
+	anchor.anchorBlock(block)
+}
+
+// AnchorByBlockNum anchors the block of given block number to KAS by API.
+func (anchor *Anchor) AnchorByBlockNum(blkNum uint64) error {
+	block := anchor.bc.GetBlockByNumber(blkNum)
```

Comment:
As far as I know, most of Klaytn APIs get `blockNumber` in methods of API packages considering other keywords like "latest". 
I think `anchor.anchorBlock` is enough for this package.

Reply:
It makes sense. I will remove this function and use `anchorBlock` directly in API.

Reply:
reflected